src/vivarium_nih_us_cvd/components/healthcare_utilization.py

```diff
@@ -272,17 +272,9 @@ def determine_followups_ldlc(self, pop_visitors: pd.DataFrame) -> pd.Index:
         visitors_high_ldlc = visitors.intersection(
             measured_ldlc[measured_ldlc >= data_values.LDLC_THRESHOLD.LOW].index
         )
-        visitors_on_ldlc_medication = visitors.intersection(
-            pop_visitors[
-                pop_visitors[data_values.COLUMNS.LDLC_MEDICATION]
-                != data_values.LDLC_MEDICATION_LEVEL.NO_TREATMENT.DESCRIPTION
-            ].index
-        )
-        # Schedule those on ldlc medication and have high ldlc or those not on
-        # ldlc medication but have high ASCVD and ldlc
-        needs_followup = (
-            visitors_on_ldlc_medication.union(visitors_high_ascvd)
-        ).intersection(visitors_high_ldlc)
+
+        # Schedule those with high ldlc and high ASCVD
+        needs_followup = visitors_high_ascvd.intersection(visitors_high_ldlc)
```

Comment:
Add a comment about why it's no longer necessary to check for medication status.